Kudu.Core/Scripts/kudusync

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-"$0.NET.exe" "$@"
+mono "$0.NET.exe" "$@"
```

Comment:
I guess it must be used when users generate a custom deployment script and choose bash for the type. We need to test this.